.gitignore

```diff
@@ -161,3 +161,6 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+
+# OS X
+.DS_Store
```

Comment:
Instead of adding here, it's a good idea to configure global gitignore on your own machine, see https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer and https://github.com/github/gitignore/blob/main/Global/macOS.gitignore

Reply:
Fair, although a "too good" gitignore is not much of a problem.